pipelines/capture/serpro/tasks.py

```
@@ -45,35 +46,42 @@ def extract_data():
+            )
+
             jdbc.execute_query(query)
             columns = jdbc.get_columns()
-        except Exception as e:
-            log(f"Erro ao executar query ou obter colunas: {str(e)}")
-            raise
-        output = io.StringIO()
-        csv_writer = csv.writer(output)
 
-        csv_writer.writerow(columns)
+            temp_file = tempfile.NamedTemporaryFile(mode="w+", delete=False, suffix=".csv")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)